emily/api-definition/package.json

```diff
@@ -1,10 +1,11 @@
 {
+  "name": "emily-api",
   "scripts": {
     "build-openapi": "smithy build",
     "clean-openapi": "smithy clean",
-    "build-rs": "npx @openapitools/openapi-generator-cli generate -i ../../.generated-sources/openapi/openapi/Emily.openapi.json -g rust -c rust-config.json -o ../../.generated-sources/emily",
+    "build-rs": "npx @openapitools/openapi-generator-cli generate -i ../../.generated-sources/smithy/openapi/openapi/Emily.openapi.json -g rust -c rust-config.json -o ../../.generated-sources/emily",
     "build": "npm run build-openapi && npm run build-rs",
-    "clean": "rm -rf node_modules  && npm run clean-openapi"
+    "clean": "rm -rf node_modules && npm run clean-openapi"
```

Comment:
```suggestion
    "clean": "rm -rf node_modules && npm clean-openapi"
```

Reply:
Why did that work before?

Reply:
Oh I have a typo but your code did it right (my suggestion is `npm clean..` but I meant `pnpm clean`.

With NPM you have to do `npm run X` for any script in your package.json, but pnpm automatically pulls all your script names into the "top level". So it works with pnpm but not npm.